Mlem/Logic/Time Parser.swift

```diff
@@ -7,29 +7,29 @@
 
 import Foundation
 
-func convertResponseDateToDate(responseDate: String) -> Date
-{
-    var convertedDate: Date?
-    
-    AppConstants.dateFormatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSSSSS"
-    AppConstants.dateFormatter.timeZone = .gmt
-    
-    convertedDate = AppConstants.dateFormatter.date(from: responseDate)
-    
-    if convertedDate == nil /// Sometimes, the API returns a different date format. If that happens, try the alternative formatter
-    {
-        AppConstants.dateFormatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ss"
-        convertedDate = AppConstants.dateFormatter.date(from: responseDate)
-    }
-    
-    #warning("TODO: Remove this in favor of figuring out WTF is crashing it")
-    if convertedDate == nil
-    {
-        convertedDate = Date()
-    }
-    
-    return convertedDate!
-}
+//func convertResponseDateToDate(responseDate: String) -> Date
```

Comment:
Maybe remove entirely?

Reply:
addressed in 60d6229fce1469f7e2ee636f593e55e7796daf8c